<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiscFltWidget</class>
 <widget class="QWidget" name="MiscFltWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QCheckBox" name="sortRteCheck">
       <property name="text">
        <string>Sort Routes</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="transformCheck">
         <property name="toolTip">
          <string>Convert routes, waypoints and tracks to different types.</string>
         </property>
         <property name="whatsThis">
          <string>This filter can be used to convert GPS data between different data types.

Some GPS data formats support only some subset of waypoints, tracks, and routes. The transform filter allows you to convert between these types. For example, it can be used to convert a pile of waypoints (such as those from a CSV file) into a track or vice versa. </string>
         </property>
         <property name="text">
          <string>Transform</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="transformCombo">
         <property name="toolTip">
          <string>Type of transformation. </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="deleteCheck">
         <property name="toolTip">
          <string>Delete original data after transform to prevent duplicated data. </string>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Nuke (Remove) Data Types</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QCheckBox" name="nukeRoutes">
          <property name="text">
           <string>Routes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="nukeTracks">
          <property name="text">
           <string>Tracks</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="nukeWaypoints">
          <property name="text">
           <string>Waypoints</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>By</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="sortWptBy">
         <item>
          <property name="text">
           <string>Description</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GCID</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Name</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Time</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="sortWptCheck">
       <property name="text">
        <string>Sort Waypoints</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>By</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="sortRteBy">
         <item>
          <property name="text">
           <string>Description</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Name</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Number</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QCheckBox" name="swapCheck">
       <property name="toolTip">
        <string>Swap Longitude and Latitudes for badly formatted data formats.</string>
       </property>
       <property name="whatsThis">
        <string>Simple filter to swap the coordinate values (latitude and longitude) of all points. This can be helpful for wrong defined/coded data. Or if you think, you can use one of our xcsv formats, but latitude and longitude are in opposite order. </string>
       </property>
       <property name="text">
        <string>Swap Coordinates</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Misc. Filters</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="sortTrkCheck">
       <property name="text">
        <string>Sort Tracks</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>By</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="sortTrkBy">
         <item>
          <property name="text">
           <string>Description</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Name</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Number</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
